lalrpop/src/lr1/codegen/parse_table.rs

```diff
@@ -238,6 +238,9 @@ impl<'ascent, 'grammar, W: Write> CodeGenerator<'ascent, 'grammar, W, TableDrive
             rust!(self.out, "{}({}),", name, ty);
         }
 
+        let error_type = self.types.parse_error_type();
+        rust!(self.out, "Error({})", error_type);
```

Comment:
Seems like it should be fine, since we normally prefix the variant names with `Nt` or `Term` in `variant_name_for_symbol`. Maybe add a comment about this?